### ISSUE TITLE
Alter available stacks in Font drop down

### ISSUE THREAD

**Issue by Helmi** [soflyy/oxygen-bugs-and-features#638]
It would be great if we could have a way to alter the available font stacks in the font dropdown in anyway. Maybe there is one already? A way to do it with a little php function would be good enough for me.

By altering the font stack I mean to have the ability to define my own font stack for example to do something like:

```
-apple-system, BlinkMacSystemFont, “Roboto”, “Droid Sans”, “Helvetica Neue”, Helvetica, Arial, sans-serif
```

Which is only one example of a currently widely used Stack of system fonts. 

I know that we can overrride the fonts via css already but it would be more oxygen like to just have it there as a font stack in the dropdown, right?

Thank you!

**Comment by wpinfos** [comment]
Would like to use system stack as well!

**Comment by Helmi** [comment]
This would also help to simply add a custom font to the list without the need to use the Elegant Custom Fonts plugin (which has some bugs/limitations in terms of number of weights).

**Comment by ayhanmalkoc** [comment]
+1

**Comment by AztecInternet** [comment]
Yeah I’d like this. After speeding up my site I now have to wait half a second on every page load with my site rendering Times New Roman before Google throws me OpenSans. It sucks. I’m happy to use a system font stack if it’s possible.

Please make it an option in the Global section.

**Comment by jvorld** [comment]
+1

**Comment by swinggraphics** [comment]
The ability to specify a fallback font stack is critical, too. When Oxygen generates `body { font-family: 'Lato' }`, that is insufficient. It needs to be `'Lato, Arial, sans-serif`… or whatever. We need a field to manually enter a font stack below the global settings.

Display font
<select>
Fallback
<text>

Text font
<select>
Fallback
<text>

**Comment by guyinpv** [comment]
Would really appreciate this. At least put in an option to say "none" so we can easily use custom css, or even have it switch to a plain textbox so we can free-form type it in, something! Before this issue is two years old!

**Comment by swinggraphics** [comment]
Commenting again as this is coming up for me again and again. Why isn't there a simple filter for the Font Family list? Even better: a page of the Oxygen settings that lets us enter a string. I just need to be able to specify a font stack. This is needed both for system fonts and services like Fonts.com.

And again, it's also very necessary for Google fonts—we need to specify fallbacks.

**Comment by d19dotca** [comment]
Similar request which could almost be merged with this one: https://github.com/soflyy/oxygen-bugs-and-features/issues/141

I'm definitely in favour of something like this, as I like to personally override the font options with a default set which uses system fonts as preferred first. 

```-apple-system,"system-ui",Segoe UI,roboto,helvetica,arial,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol```

Right now there's no way other than Stylesheets to set that, which renders the whole Global Font functionality moot once set.
